test/passes/post-emscripten.wast

```diff
@@ -57,4 +60,70 @@
       )
     )
   )
+  (func $exc
+    (call $invoke_vif
+      (i32.const 3) ;; other_safe()
+      (i32.const 42)
+      (f32.const 3.14159)
+    )
+    (call $invoke_vif
+      (i32.const 4) ;; other_unsafe()
+      (i32.const 55)
+      (f32.const 2.18281828)
+    )
+    (call $invoke_vif
+      (i32.const 5) ;; deep_safe()
+      (i32.const 100)
+      (f32.const 1.111)
+    )
+    (call $invoke_vif
+      (i32.const 6) ;; deep_unsafe()
+      (i32.const 999)
+      (f32.const 1.414)
+    )
+    (call $invoke_vif
+      (i32.add (i32.const 1) (i32.const 1)) ;; nonconstant
+      (i32.const 42)
+      (f32.const 3.14159)
+    )
+  )
+  (func $other_safe (param i32) (param f32)
+  )
+  (func $other_unsafe (param i32) (param f32)
+    (drop
+      (call $Math_pow
+        (f64.const 1)
+        (f64.const 3)
+      )
+    )
+  )
+  (func $deep_safe (param i32) (param f32)
+    (call $other_safe (unreachable) (unreachable))
+  )
+  (func $deep_unsafe (param i32) (param f32)
+    (call $other_unsafe (unreachable) (unreachable))
+  )
+)
+(module ;; no invokes
+  (func $call
+    (call $call)
+  )
+)
```

Comment:
What does this module test?

Reply:
A module that has no invokes at all. Maybe not much of a corner case, but just to be safe...